### DISCUSSION TITLE
✅ How to forward multiple ports?

### DISCUSSION THREAD

**Discussion by johnsonw** [isontheline/pro.webssh.net#443] [Q&A] [answered]
Hello! I have a raspberry pi with vs [code-server](https://github.com/cdr/code-server) installed so that I can write software over my browser on my iPad. It works great, but there are a couple of other ports I would like to forward. On my laptop, I simply use the following configuration in my ssh config file:

```
Host code-server
    HostName 192.168.1.41
    User ubuntu
    LocalForward 8080 127.0.0.1:8080
    LocalForward 8000 127.0.0.1:8000
```

When I try to configure this in webssh it seems that I can forward one or the other, but not both. I've tried separating them with a comma:
```
8000:127.0.0.1:8000,8080:127.0.0.1:8080
```
but this doesn't seem to work. Does anyone know how to forward more than one port? Thanks in advance.

Regards,

Will

**Comment by isontheline** [comment, marked as answer]
Hello @johnsonw 👋

Yes it's a known limitation, an issue is already open.
You can track it here : https://github.com/isontheline/pro.webssh.net/issues/220

I hope to ship it on WebSSH 17 - Shell on November

**Reply by johnsonw** [reply]
Sounds good @isontheline. Do you need any help with the PR?

**Reply by isontheline** [reply]
WebSSH repository is private so public PR can't be made but, I will need help for testing on TestFlight when ready thank you a lot!
